### PR TITLE
fix: add stubbed date to Calendar for Happo tests

### PR DESCRIPTION
**Pull request by separatio**
No ticket.

### Description

Multiple PRs have this fix. Decided to fix it separately quickly and force merge it.

### How to test

- Happo tests pass

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Development checks

- [x] Ensure that tests pass
- [x] Self reviewed
- [x] Covered with tests

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

File: cypress/component/Calendar.spec.tsx [hunk 1]
```diff
@@ -11,6 +11,12 @@ const TestCalendar = (props: ComponentProps<typeof Calendar>) => (
 )
 
 describe('Calendar', () => {
+  // cy.clock() fixes the time so that the screenshots
+  // in happo are not affected by server timezone
+  before(() => {
```
Comment (dmaklygin) [inline cypress/component/Calendar.spec.tsx:16]: `beforeEach` maybe?

**Review by dmaklygin — APPROVED** [review]

**Review by denieler — COMMENTED** [review]
I guess a similar change has been already done in #2723

File: cypress/component/Calendar.spec.tsx (hunk 1 above)
Comment (separatio) [reply cypress/component/Calendar.spec.tsx:16]: Considering the documentation, yeah, I'll use `beforeEach` instead.
https://docs.cypress.io/api/commands/clock#Yields

Should be less flaky in the future. :+1:

Comment (separatio) [reply cypress/component/Calendar.spec.tsx:16]: What I mean is, the browser state is cleared between tests automatically in Cypress, if we use before.

**Review by ozgurkececioglu — APPROVED** [review]

**Comment by separatio** [comment]
Tried fixing it faster. Contributor was faster than me. _makes sad noises and goes into the distance reconsidering life choices_

**Pull request closed without merging**